src/components/character-relationships/spec.js

```diff
@@ -0,0 +1,183 @@
+import React from 'react';
+import test from 'tape';
+import { shallow } from 'enzyme';
+import spy from 'utils/spy';
+
+import ListItem from 'material-ui/lib/lists/list-item';
+import IconButton from 'material-ui/lib/icon-button';
+import Dialog from 'material-ui/lib/dialog';
+import FlatButton from 'material-ui/lib/flat-button';
+import TextField from 'material-ui/lib/text-field';
+
+import CharacterRelationships from './';
+
+test('CharacterRelationships', t => {
+    let expected, actual;
+
+    let relationships = [
+        {
+            _id: 'the first id',
+            avatar: 'https://en.gravatar.com/userimage/10460545/28033602a20a2b3e9cd4db965c3b43c4.jpeg',
+            name: 'Chris Harrington',
+            description: 'Father'
+        },
+        {
+            _id: 'the second id',
+            name: 'Bruce Wayne',
+            description: 'Batman'
+        }
+    ];
+
+    {
```

Comment:
This is my first time using Tape and from everything I've seen, you're supposed to keep everything in one `test` block. I was initially going to create separate functions for all of the tests, but didn't want to have to add an invocation along with the separate methodl. The code blocks were a good compromise. After years of trying to keep functions relatively small, putting everything in one method was bugging me. Boils down to purely a style choice, I guess. I can remove them if it's a problem.